libnavigation-core/src/main/java/com/mapbox/navigation/core/telemetry/TelemetryEventUserFeedback.kt

```diff
@@ -0,0 +1,89 @@
+package com.mapbox.navigation.core.telemetry
+
+import android.location.Location
+import com.google.gson.Gson
+
+internal enum class FeedbackType(val feedbackType: String) {
+    FEEDBACK_TYPE_ACCIDENT("accident"),
+    FEEDBACK_TYPE_HAZARD("hazard"),
+    FEEDBACK_TYPE_ROAD_CLOSED("road_closed"),
+    FEEDBACK_TYPE_NOT_ALLOWED("not_allowed"),
+    FEEDBACK_TYPE_ROUTING_ERROR("routing_error"),
+    FEEDBACK_TYPE_MISSING_ROAD("missing_road"),
+    FEEDBACK_TYPE_MISSING_EXIT("missing_exit"),
+    FEEDBACK_TYPE_CONFUSING_INSTRUCTION("confusing_instruction"),
+    FEEDBACK_TYPE_INACCURATE_GPS("inaccurate_gps"),
+    FEEDBACK_SOURCE_REROUTE("reroute"),
+    FEEDBACK_SOURCE_UI("user")
+}
+
+internal enum class FeedbackSource(val feedbackSource: String) {
```

Comment:
This is never used.

Reply:
👍 Same as above. Like, I said, preliminary PR